### ISSUE TITLE
Question on ground truth for action recognition

### ISSUE THREAD

**Issue by tsly123** [gurkirt/realtime-action-detection#16]
Hi @gurkirt ,

Firstly, thank you for publishing your work.

Second, I have a question and I hope we can discuss it here. 

1.  In almost all recent paper on action recognition topic, they resized the original frame image from `(320*240*3)` into other sizes, for yours is `(300*300*3)`.

That makes me wonder what about the ground truth? The ground truth which is 4 number of pixel coordinates, is for original video dataset `(320*240*3)` and when resizing the image, did you also resized the ground truth into `(300*300*3)` form as well or use the original one? Because their algorithms usually based on the ground truth overlapping for evaluating the accuracy, so I think it will make a big difference in the result.
By the way, do you know which version for the ground truth they used in their work for UCF-101 24 classes and other datasets? I am asking because I found UCF-101 has 2 version of the ground truth: one is the original and the other one was remade by you for [Thumos 2015](http://www.thumos.info/download.html). This gets me confused.

2. Could you explain more about the training SSD step. What is purpose of the training step?
Your work used the pre-trained Vgg-16 on VOC (21 classes) to detect for UCF-24 (24 classes), not to detect directly the person?

Thank you.

**Comment by gurkirt** [comment]
1. Has been taken take. 
2. SSD implementation is based on https://github.com/amdegroot/ssd.pytorch, where they evaluate on VOC and I verified it.
